test/BasicNetworkTest.template.ts

```diff
@@ -17,12 +17,13 @@
 import { Network } from '@core';
 import { expect } from 'chai';
 
-export const AssertNetwork = (network: Network, expectedName: string, expectedIdentifier: number): void => {
-    it('correct predefined networks are registered', () => {
-        // Assert:
-        expect(network.name).to.be.equal(expectedName);
-        expect(network.identifier).to.be.equal(expectedIdentifier);
-    });
+export const AssertNetwork = (network: Network | undefined, expectedName: string, expectedIdentifier: number): void => {
```

Comment:
Why do some methods start with a capital letter? Should be having PascalCase for classes only, for this instance, it should be `assertNetwork`?

Reply:
It should be assertNetwork, I can change it in this pr if no problem.

Reply:
Renamed